css/style.css

```diff
@@ -0,0 +1,251 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 60px;
+    padding: 15px;
+    box-shadow: 0 1px 5px #eee;
+    background-color:#fff;
+  }
+
+.left-nav {
+    display: flex;
+    align-items: center;
+}
+   
+.left-nav #menu {
+    padding: 0 7px;
+    cursor: pointer;
+}
+
+.center-nav {
+    display: flex;
+}
+   
+.center-nav form {
+    display: flex;
+    border: 1px solid #ddd;
+    height: 45px;
+}
+
+.center-nav input {
+    width: 600px;
+    padding:10px;
+    border: 0;
+    height: 100%;
+    border-radius: 2px 0 0 2px
+  }
+  
+  input:focus {
+    outline: none;
+    border: 1px solid #ddd;
+  }
+
+  .center-nav button {
+    height: 100%;
+    width: 60px;
+    border: none;
+  }
+   
+  #mic{
+    margin-top: 10px;
+  }
+
+  i {
+    color: rgb(100, 100, 100);
+    padding: 0 7px;
+    cursor: pointer;
+  }
+
+
+
+
+  /* Main  */
+
+
+  
```

Comment:
Please it would be much better if you don't  use too much spacing in your code, to much space can lead to linters errors and reading your code becomes very difficult